ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Report an issue
+about: Create a report about an issue you found in the Mattermost
+---
+
+
+Per Mattermost guidelines, GitHub issues are for bug reports:
+<https://handbook.mattermost.com/contributors/contributors/ways-to-contribute/>.
+
+For troubleshooting see: https://forum.mattermost.com/.  For feature proposals
+see: https://mattermost.com/suggestions/
+
+If you've found a bug--something appears unintentional--please follow these
+steps:
+
+1. Confirm you’re filing a new issue. [Search existing tickets in
+   Jira](https://mattermost.atlassian.net/jira/software/c/projects/MM/issues/)
+   to ensure that the ticket does not already exist.
+2. Confirm your issue does not involve security. Otherwise, please see our
+   [Responsible Disclosure
+   Policy](https://mattermost.com/security-vulnerability-report/).
+3. [File a new issue](https://github.com/mattermost/mattermost/issues/new)
+   using the format below. Mattermost will confirm steps to reproduce and file
+   in Jira, or ask for more details if there is trouble reproducing it. If
+   there's already an existing bug in Jira, it will be linked back to the
+   GitHub issue so you can track when it gets fixed.
+
+#### Summary Bug report in one concise sentence
+
+#### Steps to reproduce How can we reproduce the issue (what version are you
+using?)
+
+#### Expected behavior Describe your issue in detail
+
+#### Observed behavior (that appears unintentional) What did you see happen?
+Please include relevant error messages and/or screenshots.
+
+#### Possible fixes If you can, link to the line of code that might be
+responsible for the problem
```

Comment:
Something happened with the formatting here

Reply:
good catch, updated